src/plugins/intel_cpu/src/emitters/tpp/x64/utils.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "libxsmm.h"
+#include "emitters/utils.hpp"
+#include "openvino/core/type/element_type.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace tpp {
+namespace utils {
+
+inline libxsmm_datatype ov_to_xsmm_dtype(ov::element::Type_t elemet_type) {
+    switch (elemet_type) {
+        case ov::element::Type_t::f32 : return LIBXSMM_DATATYPE_F32;
+        case ov::element::Type_t::bf16 : return LIBXSMM_DATATYPE_BF16;
+        case ov::element::Type_t::f16 : return LIBXSMM_DATATYPE_F16;
+        case ov::element::Type_t::i8 : return LIBXSMM_DATATYPE_I8;
+        case ov::element::Type_t::u8 : return LIBXSMM_DATATYPE_U8;
+        default:
+            OV_CPU_JIT_EMITTER_THROW("Attempt to convert unsupported ov data type");
```

Comment:
Minor: let's print the unsupported type in exception message:
```suggestion
            OV_CPU_JIT_EMITTER_THROW("Attempt to convert unsupported ov data type: ", element_type);
```

Reply:
done in [668504d](https://github.com/openvinotoolkit/openvino/pull/28304/commits/668504db46e95358a60def59715496475d6b0f86), thanks Vladislav!